gdk/commands/test/config/InitConfiguration.py

```diff
@@ -10,6 +12,23 @@ def __init__(self, _args) -> None:
     def _get_otf_version(self):
```

Comment:
how are we getting OTF? Instead of only validating the semver, why not just check to see if the version requested actually exists? `100.0.0` is valid semver, but not a valid OTF version (at least not currently).

Reply:
Updated code checks if the version tag exists.

Customers will have to provide the OTF version as they find it in the [releases](https://github.com/aws-greengrass/aws-greengrass-testing/releases). GDK will append `-SNAPSHOT` to that version for now as that's how all the testing jars have been so far.